File: AppGenerateFiles/dbo/Model/Transaction_Ingresos.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaction_Ingresos : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? numero_ingreso { get; set; }
       public Double? monto { get; set; }
       public DateTime? fecha { get; set; }
       public string? descripcion { get; set; }
       public string? nombre { get; set; }
       public int? que { get; set; }
       public string? anulado { get; set; }
       public string? observaciones { get; set; }
       public Double? tzcambio { get; set; }
       public Double? total { get; set; }
       public DateTime? fanulado { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Transaction_Ingresos.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
class Transaction_Ingresos extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ numero_ingreso;
   /**@type {Number}*/ monto;
   /**@type {Date}*/ fecha;
   /**@type {String}*/ descripcion;
   /**@type {String}*/ nombre;
   /**@type {Number}*/ que;
   /**@type {String}*/ anulado;
   /**@type {String}*/ observaciones;
   /**@type {Number}*/ tzcambio;
   /**@type {Number}*/ total;
   /**@type {Date}*/ fanulado;
}
export { Transaction_Ingresos }
